operations/pyroscope/helm/pyroscope/README.md

```diff
@@ -94,4 +94,4 @@
 | serviceMonitor.tlsConfig | string | `nil` | ServiceMonitor will use these tlsConfig settings to make the health check requests |
 
 ----------------------------------------------
-Autogenerated from chart metadata using [helm-docs v1.8.1](https://github.com/norwoodj/helm-docs/releases/v1.8.1)
+Autogenerated from chart metadata using [helm-docs v1.14.2](https://github.com/norwoodj/helm-docs/releases/v1.14.2)
```

Comment:
One last bit to make the linter happy – we should either update the helm-docs version in [the makefile](https://github.com/grafana/pyroscope/blob/main/Makefile#L397) or specify it manually here (1.8.1) or just run `make helm/docs`